components/06-examples/checken.hbs

```diff
@@ -1,27 +1,30 @@
 {{> '@header' }}
 
 <main>
-  <h1>Checken</h1>
   <div class="row">
-    <div class="col-md-9">
-      <h2>Werkzaamheden checken</h2>
-      <h3>Bouwen van een dakkapel</h3>
-
-      {{render '@highlight-box' }}
+    <div class="col-xs-12">
+      <h1>Vergunningcheck</h1>
+        {{render '@steps'}}
     </div>
-    <div class="col-md-3">
-      {{#progressBlock }}
-        <h2>Tussenresultaten</h2>
-        <ul class="list-unstyled">
-          {{#each checkSideListItems }}
-            <li>
-              {{ this }}
-            </li>
-          {{/each}}
-        </ul>
-      {{/progressBlock}}
+  </div>
+  <div class="row">
+    <div class="col-md-12">
```

Comment:
Hier zet je een `col-md-12` en op regel 5 een `col xs-12`. Het responsive gedrag is nu hetzelfde maar ik zie toch liever twee dezelfde classes